### ISSUE TITLE
[Feature Request] Ability to automatically save search results in HTML as the search page is updated

### ISSUE THREAD

**Issue by monsippi** [FooSoft/yomichan#1739]
I've been trying to figure out a way to automatically fetch the results of the search.html tab as I copy a sentence to the clipboard. Essentially, I'd like a scraper to automatically detect the search page being updated (which happens due to the clipboard update) and scrape the results to display in an external program. I've tried working around it but I'm having trouble with the "moz-extension" tab. I think it should be fairly straightforward to implement something like this.

**Comment by toasted-nutbread** [comment]
Do you want the raw HTML that is generated, or would it be useful to get the raw data (JSON) that is used to generate the HTML?

**Comment by monsippi** [comment]
Either would be fine.

**Comment by toasted-nutbread** [comment]
I don't think there's currently a way, unless you do something ugly involving some sort of UI automation. This kind of falls into the category of #1649, and that issue is requesting that the raw JSON be exposed, hence the ask.

**Comment by monsippi** [comment]
Right. It's possible to write a browser extension that monitors the search page and parses it when the page updates, but I kinda gave when I got stuck at the Firefox Tabs API because I couldn't run the script on Yomichan's tab because the "moz-extension" scheme can't be used in the matching. 

A workaround like this could be messy. For now I'm tabbing back to the browser every time I copy to the clipboard. 

I'll keep an eye on the other issue you mentioned. You can close this one if you want to.
